frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/review-tab.scss

```diff
@@ -5,3 +5,56 @@
 .kubevirt-create-vm-modal__review-tab-lower-section {
   margin-top: 20px;
 }
+
+.kubevirt-create-vm-modal__review-tab-section-container {
```

Comment:
I think we can adjust the class names to be a bit shorter :) 
`kv-create-vm__review__section` wdyt?

Reply:
I'm all for it, but maybe in a future PR. This follows the convention that's used through the plugin and I think if we're going to change it, we should change it everywhere at the same time.